src/helpers.js

```diff
@@ -202,14 +202,20 @@ function updateField(field, requiredFields, node, formValues, logic, config) {
     field.isVisible = true;
   }
 
-  const updateValues = (fieldValues) =>
+  const updateValues = (fieldValues) => {
+    ['const', 'default', 'minimum', 'maximum', 'minLength', 'maxLength'].forEach((key) => {
```

Comment:
todo: I don't need to be explicit about the downsides of having a hardcoded list of keys right? So the best we can do for now is to extract this to a function/variable and add a little but important comment about it.

p.s. It's missing `readOnly`,  `description`, and `statement`, which are very common too.

Reply:
exactly, this was thrown together, will improve! ... i think it causes other problems for descriptions, need to investigate exactly what happens